### DISCUSSION TITLE
How do embeddings work when we update a preexisting file or create a new file?

### DISCUSSION THREAD

**Discussion by avataraustin** [brianpetro/obsidian-smart-connections#203] [Q&A] [answered]
I am wondering how often my vault gets processed for embeddings. I just added some content to a preexisting note file and then later I queried chatgpt to search my notes for an obviously worded question but unfortunately it was completely unaware of the updated note.  Does Smart Connections periodically update embeddings? Can it be triggered? Is there something I need to do to get it to update embeddings?  Thanks.

**Comment by brianpetro** [comment, marked as answer]
@avataraustin there's a minimum threshold before updating. 

You can see additional logging in the console when you toggle on the ["log render" advanced settings.](https://github.com/brianpetro/obsidian-smart-connections#settings-advanced)

These logs can show you which files were embedded or skipped. 

You can force re-embedding a specific file by running the "Find Smart Connections" command on the target file.

Brian 🌴

**Reply by avataraustin** [reply]
Awesome , thank you. This app is such a game changer.
